qiskit/providers/aer/extensions/snapshot_probabilities.py

```diff
@@ -0,0 +1,46 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals
+
+from qiskit import QuantumCircuit
+from qiskit.circuit.quantumregister import QuantumRegister
+from qiskit.providers.aer.extensions import Snapshot
+
+class SnapshotProbabilites(Snapshot):
+
+    def __init__(self,
+                 label,
+                 num_qubits,
+                 snapshot_type='probabilities',
+                 num_clbits=0,
+                 params=None,
+                 variance=False):
+
+        super().__init__(label, snapshot_type, num_qubits, num_clbits, params)
+
+        if variance:
```

Comment:
This needs to come above the call to `super()` so that it is used in that method.